keras_nlp/models/deberta/deberta_backbone.py

```diff
@@ -0,0 +1,274 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DeBERTa backbone models."""
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_nlp.models.deberta.deberta_encoder import DebertaEncoder
+from keras_nlp.utils.python_utils import classproperty
+
+
+def deberta_kernel_initializer(stddev=0.02):
+    return keras.initializers.TruncatedNormal(stddev=stddev)
+
+
+class RelativeEmbedding(keras.layers.Layer):
```

Comment:
Let's split this into a separate file, in keeping with what you did for the other layers.